pages/integrations/inputs/okta_input.rst

```diff
@@ -0,0 +1,32 @@
+.. _okta_input:
+
+*********************
+Okta Log Events Input
+*********************
+
+.. attention:: This is an Enterprise Integrations feature and is only available since Graylog version 3.3 and thus requires an Enterprise license.
```

Comment:
```suggestion
.. attention:: This is a Graylog Enterprise feature and is only available since Graylog version 3.3.  A valid Graylog Enterprise license is required.
```